__tests__/topics.test.js

```diff
@@ -52,12 +50,51 @@ describe("Topics api testing", () => {
         expect(topics).toHaveLength(3);
 
         expect(topics[0].slug).toBe("mitch");
-        expect(topics[1].slug).toBe('cats');
-        expect(topics[2].slug).toBe('paper');
+        expect(topics[1].slug).toBe("cats");
+        expect(topics[2].slug).toBe("paper");
 
         expect(topics[0].description).toBe("The man, the Mitch, the legend");
-        expect(topics[1].description).toBe('Not dogs');
-        expect(topics[2].description).toBe('what books are made of');
+        expect(topics[1].description).toBe("Not dogs");
+        expect(topics[2].description).toBe("what books are made of");
+      });
+  });
+});
+
+describe("Articles api testing", () => {
+  test("GET the status of /api/articles/1", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+
+  test("GET the response and status of /api/articles/:article_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
```

Comment:
I think here you can be more specific and check the specific id, especially as you have requested that one, you wouldn't want it to send back id 12 for example